libs/components/src/lib/menu-item/menu-item.spec.ts

```diff
@@ -324,4 +335,90 @@ describe('vwc-menu-item', () => {
 
 		});
 	});
+
+	describe('keydown', () => {
+		beforeEach(async function () {
+			menuElement = (await fixture(
+				`<${MENU_TAG} open>
+					<${COMPONENT_TAG} id="menuitem" text="Menu item 1">
+						<${MENU_TAG} slot="submenu">
+							<${COMPONENT_TAG} text="Menu item 1.1"></${COMPONENT_TAG}>
+							<${COMPONENT_TAG} text="Menu item 1.2"></${COMPONENT_TAG}>
+							<${COMPONENT_TAG} text="Menu item 1.3"></${COMPONENT_TAG}>
+						</${MENU_TAG}>
+					</${COMPONENT_TAG}>
+				</${MENU_TAG}>`
+			)) as Menu;
+
+			await elementUpdated(menuElement);
+		});
+
+		it('should expand first menuitem on ArrowRight and close on ArrowLeft', async () => {
+			const menuitem = menuElement.querySelector('#menuitem') as MenuItem;
+
+			menuitem.focus();
+			menuitem.dispatchEvent(new KeyboardEvent('keydown', { key: 'ArrowRight' }));
+
+			await elementUpdated(menuElement);
+			expect(menuitem.expanded).toEqual(true);
+
+			menuitem.dispatchEvent(new KeyboardEvent('keydown', { key: 'ArrowLeft' }));
+
+			await elementUpdated(menuElement);
+			expect(menuitem.expanded).toEqual(false);
+		});
+		
+		it('should expand first menuitem on Enter/Space', async () => {
+			const menuitem = menuElement.querySelector('#menuitem') as MenuItem;
+
+			menuitem.focus();
+			menuitem.dispatchEvent(new KeyboardEvent('keydown', { key: 'Enter' }));
+
+			await elementUpdated(menuElement);
+			expect(menuitem.expanded).toEqual(true);
+
+			menuitem.dispatchEvent(new KeyboardEvent('keydown', { key: 'ArrowLeft' }));
+
+			await elementUpdated(menuElement);
+			expect(menuitem.expanded).toEqual(false);
+
+			menuitem.focus();
+			menuitem.dispatchEvent(new KeyboardEvent('keydown', { key: ' ' }));
+
+			await elementUpdated(menuElement);
+			expect(menuitem.expanded).toEqual(true);
+		});
```

Comment:
As this is basically testing the same thing with different keys, maybe we can write it like this instead?

```suggestion
		it.each(['Enter', ' '])('should expand first menuitem when "%s" is pressed', async (key) => {
			const menuitem = menuElement.querySelector('#menuitem') as MenuItem;

			menuitem.focus();
			menuitem.dispatchEvent(new KeyboardEvent('keydown', { key }));

			await elementUpdated(menuElement);
			expect(menuitem.expanded).toEqual(true);
		});
```